.gitignore

```diff
@@ -1,13 +1,66 @@
+# iOS / Apple
+# ===========
 .DS_Store
+ehthumbs.db
+Icon?
+Thumbs.db
+config/env/local.js
+
+# Node and related ecosystem
+# ==========================
 .nodemonignore
 .sass-cache/
-*.log
```

Comment:
@ilanbiala can you be more clear? we have a *.log at the end of the .gitignore and that should cover all such files as npm-debug.log, and I'll now commit to remove access.log too.